dagrunner/tests/utils/test_function_to_parse.py

```diff
@@ -12,6 +12,24 @@
 CALLING_MOD = os.path.basename(sys.argv[0])
 
 
+def assert_help_str(help_str, tar):
+    # Remove line wraps for easier comparison
+    formatted_help_str = []
+    help_strs = help_str.split("\n")
+    for line in help_strs:
+        if line.startswith("   "):
+            # more than 2 space indent
+            formatted_help_str[-1] += f" {line.lstrip()}"
+        else:
+            formatted_help_str.append(line)
+    help_str = "\n".join(formatted_help_str)
+
+    if "optional arguments:" in help_str:
+        # older versions of argparse use "optional arguments: instead of options:"
+        tar = tar.replace("options:", "optional arguments:")
+    assert help_str == tar
```

Comment:
Supporting old and new argparse versions (not sure when this exactly changed right now).  Was a cause of failures on CI and local suceeds.